### ISSUE TITLE
consider shortening names for functions under internal/objects

### ISSUE THREAD

**Issue by skriss** [projectcontour/contour-operator#222]
Now that the helpers for different object types have each moved into their own package, some of the function names are redundant, e.g. `clusterrole.EnsureClusterRole`, `clusterrole.CurrentClusterRole`, etc. It'd help readability to simplify some of these names. Here's a quick PoC of what some shortened names could look like:

https://github.com/projectcontour/contour-operator/compare/main...skriss:exp-internal-objects-naming?expand=1

If folks think that pattern reads better, I can go through and make all the changes.

**Comment by sunjayBhatia** [comment]
+1 fan of that pattern and reducing duplication

**Comment by youngnick** [comment]
I agree that we should try and reduce stuttering when reading the actual call sites.
